test/python/fe_api/causal_conv1d_bulk/test_causal_conv1d_bulk_backward_contract_unit.py

```
@@ -0,0 +1,361 @@
+        raise AssertionError(f"unhandled case {case}")
+    api = api_class(x, weight, torch.zeros_like(x), sample_bias=bias)
+    monkeypatch.setattr(backward, "cutedsl_state", lambda: (True, None))
+
+    with pytest.raises(ValueError, match="Bias"):
+        api.check_support()
+
+
+def test_backward_rejects_non_tensor_bias():
+    _, _, api_class, _ = _load_backward()
+    x = torch.zeros(1, 4, 8, dtype=torch.bfloat16)
+    with pytest.raises(TypeError, match="sample_bias must be a torch.Tensor or None"):
```

Ruff (0.16.2)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)